agent/relay-agent.c

```diff
@@ -173,6 +173,39 @@ hwangsae_relay_agent_edge_interface_handle_stop (Hwangsae1DBusEdgeInterface *
   return TRUE;
 }
 
+static gboolean
+hwangsae_relay_agent_edge_interface_handle_change_parameters (
+    Hwangsae1DBusEdgeInterface * object, GDBusMethodInvocation * invocation,
+    gchar * arg_id, gint arg_width, gint arg_height, gint arg_fps,
+    gpointer user_data)
+{
+  ChamgeReturn ret = CHAMGE_RETURN_OK;
+  g_autofree gchar *cmd = NULL;
+  g_autofree gchar *response = NULL;
+  GError *error = NULL;
+  HwangsaeRelayAgent *self = (HwangsaeRelayAgent *) user_data;
+
+  cmd =
+      g_strdup_printf
+      ("{\"to\":\"%s\",\"method\":\"streamingChangeParameters\", "
+      "\"params\": {\"width\":%d, \"height\":%d, \"fps\": %d}}",
+      arg_id, arg_width, arg_height, arg_fps);
+
+  g_debug ("hwangsae_relay_agent_edge_interface_handle_start, cmd %s", cmd);
```

Comment:
The trace should say `hwangsae_relay_agent_edge_interface_handle_change_parameters`

Reply:
Ah, good catch.